osu.Game.Rulesets.Osu/Skinning/Legacy/LegacySpinner.cs

```diff
@@ -127,5 +129,33 @@ protected override void Dispose(bool isDisposing)
             if (DrawableSpinner != null)
                 DrawableSpinner.ApplyCustomUpdateState -= UpdateStateTransforms;
         }
+
+        /// <summary>
+        /// A <see cref="Container"/> simulating osu!stable's absolute screen-space,
+        /// for perfect placements of legacy spinner components with legacy coordinates.
+        /// </summary>
+        protected class LegacyCoordinatesContainer : Container
+        {
+            /// <summary>
+            /// An offset that simulates stable's spinner top offset,
+            /// for positioning some legacy spinner components perfectly as in stable.
+            /// (e.g. 'spin' sprite, 'clear' sprite, metre in old-style spinners)
+            /// </summary>
+            public static readonly float SPINNER_TOP_OFFSET = (float)Math.Ceiling(45f * SPRITE_SCALE);
```

Comment:
what's the point of declaring this constant inside this class, but then only ever using it from the *outer* class?

Reply:
because this is the class where it should be used on, wasn't sure where to place it at first, but `SPINNER_TOP_OFFSET` should never be used without the container anyways. can be moved to `LegacySpinner` if the whole container shall be inlined.

Reply:
I'd still rather have it a public constant on the spinner and not some inner container.